Database/compiler.lua

```diff
@@ -980,13 +980,6 @@ function QuestieDBCompiler:GetDBHandle(data, pointers, skipmap, keyToRootIndex,
     return handle
 end
 
-function QuestieDBCompiler:GetVersionString() -- todo: better place
-    local _,ver = GetAddOnInfo("Questie")
-    if not ver then
-        _,ver = GetAddOnInfo("QuestieDev-master")
-    end
-    -- todo: better regex for this
-    ver = string.sub(ver, 32)
-    ver = string.sub(ver, 0, string.find(ver, "|")-1)
-    return ver
+function QuestieDBCompiler:GetVersionString()
```

Comment:
This can completely be replaced with `QuestieLib:GetAddonVersionString()`. QuestieLib needs to be loaded on top of the file though.

Reply:
`QuestieDBCompiler:GetVersionString()` returns `major.minor.patch` (originally parsed from the title, now just grabbed from the Version defined in toc).
`QuestieLib:GetAddonVersionString()` however returns `"v" .. tostring(major) .. "." .. tostring(minor) .. "." .. tostring(patch) .. buildType`.

Though `QuestieDBCompiler:GetVersionString()` is only used in
https://github.com/AeroScripts/QuestieDev/blob/ed756defc2a8eaaa81fa75274945bc9d69497fcb/Database/compiler.lua#L634
and
https://github.com/AeroScripts/QuestieDev/blob/ed756defc2a8eaaa81fa75274945bc9d69497fcb/Modules/QuestieEventHandler.lua#L147

Does the version string format not matter as long as it is consistent across those two lines?

Reply:
Yeah as long as these lines are consistent it doesn't matter 👍